### ISSUE TITLE
Packaging tasks should avoid using Newtonsoft.Json

### ISSUE THREAD

**Issue by ericstj** [dotnet/arcade#2763]
We should switch to the new json reader/writer APIs.  /cc @joperezr

**Comment by ericstj** [comment]
FYI I think this is needed for source build.  /cc @chcosta

**Comment by joperezr** [comment]
Makes sense. Is there a deadline for when this needs to be done? (perhaps a deadline on source build?)  I ask because I know that there is still some backlog on System.Text.Json API, and AFAIK the available API today is very low-level, so I wonder if we should wait till wrapper classes are built on top of the existing API.

**Comment by ericstj** [comment]
We determined that it is not required for source build, so it's no longer a dependency we need to drive for 3.0.  I'd see this more as dogfooding / best-practice.
